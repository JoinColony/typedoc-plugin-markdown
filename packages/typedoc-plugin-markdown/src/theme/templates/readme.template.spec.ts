import { ProjectReflection } from 'typedoc';
import { readmeTemplate } from './readme.template';

describe(`Readme Template:`, () => {
  let project: ProjectReflection;

  beforeAll(async () => {
    project = global.getProject(['reflections.ts']);
  });

  test(`should compile readme`, () => {
    const props = {
      project: project,
      model: project,
    } as any;
    expect(readmeTemplate(global.getMockContext(), props)).toMatchSnapshot();
  });
});
