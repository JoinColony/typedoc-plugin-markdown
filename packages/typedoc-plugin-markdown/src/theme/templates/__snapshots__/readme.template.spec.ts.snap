// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Readme Template: should compile readme 1`] = `
"{ breadcrumbsPartial }

# MY README

> Some block quote

<!-- some comments -->

Some text
"
`;
